app/models/external_service/external_service_receiver.rb

```diff
@@ -8,15 +8,29 @@ class ExternalServiceReceiver < ApplicationRecord
   validates_presence_of :external_service_id, :receiver_id, :response_data
 
   def show_url
-    parsed_response_data[:show_url]
+    url = parsed_response_data[:show_url]
+    if formio_submission?(url)
```

Comment:
Excited to read in your PR description that a lot of these conditionals will be going away. They're icky for sure, but if they're short-lived... :D

Reply:
Heads up that I'm pretty sure the sanger sequencing engine actually hooks into this. It probably wasn't worth the extra isolation to use this hook as opposed to something more direct, but that's what we did. So even if we get rid of surveyor/qualtrics/imserc at NU, dartmouth still relies on it.